### PR TITLE
Added more validations to create event .

### PR DESCRIPTION
**Pull request by vtserej**
1- Added can go back property (false for set 1 and 5)
2- Added validation on step 2 (User has to select a location to continue)
3- Added validation to event type not loading default on step 1 (still and issue if you go back from step 2 to one)
4- Removed commented code
5- Added save event on step 4
6- Added event description validation

### CODE REVIEW

File: TrashMobMobile/ViewModels/CreateEventViewModelNew.cs [hunk 1]
```diff
@@ -182,6 +184,18 @@ private void ValidateCurrentStep(object sender, PropertyChangedEventArgs e)
                     IsStepValid = false;
                 }
 
+                break;
+            //Step 2 validation 
+            case 1:
+                if (!string.IsNullOrEmpty(EventViewModel.Address.StreetAddress))
```
Comment (joebeernink) [inline TrashMobMobile/ViewModels/CreateEventViewModelNew.cs:190]: It's actually common to have the street address be empty (most large parks don't have addresses on the map). Best to use city instead

File: TrashMobMobile/ViewModels/CreateEventViewModelNew.cs [hunk 2]
```diff
@@ -224,6 +238,8 @@ public void SetCurrentStep(StepType step)
                 SetCurrentView();
             }
         }
+
+        CanGoBack = CurrentStep != 0 && CurrentStep != 4;
```
Comment (joebeernink) [inline TrashMobMobile/ViewModels/CreateEventViewModelNew.cs:242]: Step 1 - canGoBack false
Step 2 - canGoBack true
Step 3 - canGoBack true
Step 4 - canGoBack true (until save has been hit)
Step 5+ - canGoBack false

File: TrashMobMobile/ViewModels/CreateEventViewModelNew.cs (hunk 1 above)
Comment (vtserej) [reply TrashMobMobile/ViewModels/CreateEventViewModelNew.cs]: done

File: TrashMobMobile/ViewModels/CreateEventViewModelNew.cs (hunk 2 above)
Comment (vtserej) [reply TrashMobMobile/ViewModels/CreateEventViewModelNew.cs:242]: I asume that save action goes to next step

**Review by joebeernink — APPROVED** [review]

**Pull request merged**